services/retentionmgr/metadataDep.go

```diff
@@ -60,7 +60,7 @@ func (t *metadataDepImpl) GetDestinations() (destinations []*destinationInfo) {
 	i := 0
 	for {
 
-		log.Debug("GetDestinations: ListDestinationsByUUID on metadata")
+		log.Info("GetDestinations: ListDestinationsByUUID on metadata")
```

Comment:
I think these will cause too many logs

Reply:
not sure now .. why i decided to enable these. reverting it. :-)